### PR TITLE
fix(ci): Markdown lint comment

### PR DESCRIPTION
**Pull request by quentinDERORY**
Missing reverted PR flag

Signed-off-by: quentinDERORY <15911421+quentinDERORY@users.noreply.github.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]

**Review by themarwhal — COMMENTED** [review]
what is NR vs RPR?

**Comment by quentinDERORY** [comment]
pr/NR is pr number and pr/RPR is reverted PR, could use better naming... ^^'

File: .github/workflows/semantic-pr.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: "Semantic PR"
 
 on:  # yamllint disable-line rule:truthy
-  pull_request_target:
```
Comment (hcgatewood) [inline .github/workflows/semantic-pr.yml:5]: Why change to just pull_request? Easier to test but easier to skirt around (?)

**Review by hcgatewood — APPROVED** [review]
Thanks @quentinDERORY!

File: .github/workflows/semantic-pr.yml (hunk 1 above)
Comment (quentinDERORY) [reply .github/workflows/semantic-pr.yml:5]: it should have been pull_request from the beginning. As we are using the workflow workarround to get write permission

**Pull request merged**